tests/test_gcshcablobstore.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import os
+import sys
+import unittest
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.insert(0, pkg_root)
+
+from dss.blobstore import BlobNotFoundError  # noqa
+from dss.blobstore.gcs import GCSBlobStore  # noqa
+from dss.hcablobstore import HCABlobStore  # noqa
+from dss.hcablobstore.gcs import GCSHCABlobStore  # noqa
+from tests import utils  # noqa
+
+
+class TestGCSHCABlobStore(unittest.TestCase):
+    def setUp(self):
+        self.credentials = os.path.join(pkg_root, "gcs-credentials.json")
+        self.test_bucket = utils.get_env("DSS_GCS_TEST_BUCKET")
+        self.test_src_data_bucket = utils.get_env("DSS_GCS_TEST_SRC_DATA_BUCKET")
+        self.blobhandle = GCSBlobStore(self.credentials)
+        self.hcahandle = GCSHCABlobStore(self.blobhandle)
+
+    def tearDown(self):
+        pass
+
+    def test_verify_blob_checksum(self):
+        self.assertTrue(
+            self.hcahandle.verify_blob_checksum(
+                self.test_src_data_bucket, "test_good_source_data/0",
+                {
+                    HCABlobStore.MANDATORY_METADATA['CRC32C']['keyname']: "e16e07b9",
+                }
+            )
+        )
+
+        self.assertFalse(
+            self.hcahandle.verify_blob_checksum(
+                self.test_src_data_bucket, "test_good_source_data/1",
+                {
```

Comment:
The checksum in `tests/test_gcshcablobstore.py` is wrong.  The checksum on the actual file is correct.

I'll restructure this test to pull the actual checksums from the cloud provider (rather than from a source file). #156